<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeveloperInfo</class>
 <widget class="QMainWindow" name="DeveloperInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>231</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DeveloperInfo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btGetInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>16</y>
      <width>169</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Developer Info</string>
    </property>
   </widget>
   <widget class="QLabel" name="labInfo">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>75</y>
      <width>182</width>
      <height>57</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Application</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
